tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1854,12 +1854,19 @@ public:
    ///
    std::string GetColumnType(std::string_view column)
```

Comment:
Can you not just take a `std::string` here?

Reply:
You might remember there was a bit of a string_view hype in ROOT 2-3 years ago :smile: 
The purported advantage is that with string_view you can also pass in TStrings.
I think it's to late to change this anyway..?